server.js

```diff
@@ -94,7 +95,7 @@ module.exports = function() {
         },
         validate: {
           payload: {
-            currency: Joi.string().min(3).max(3).required(),
+            currency: Joi.any().valid(currencyFor.stripe),
```

Comment:
Not required to do that anymore since if the value doesn't match any of the values in the array then it will fail.